CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# On Github Issues and Pull Requests
```

Comment:
Yeah I really would love to avoid all requests for help here, not sure what the best way to do that is. A big flashing GIF might be a bit too off-putting, hah